src/ui/component/dateRangePicker/dateRangePicker.tsx

```diff
@@ -0,0 +1,52 @@
+import { type FC, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { DateRange } from 'react-day-picker'
+import type { DayPickerProps } from '@/ui/component/dayPicker/dayPicker'
+import { DatePicker } from '@/ui/component/datePicker/datePicker'
+import './dateRangePicker.scss'
+
+type DateRangePickerProps = Omit<DayPickerProps, 'title' | 'onSelect' | 'selected'> & {
+  onSelect: (dateRange?: DateRange) => void
+  selected?: DateRange
+}
+
+export const DateRangePicker: FC<DateRangePickerProps> = ({
+  onSelect,
+  selected,
+  ...dayPickerProps
+}) => {
+  const { t } = useTranslation('common')
+
+  const handleSelectFrom = useCallback(
+    (date?: Date) => {
+      onSelect({ from: date, to: selected?.to })
```

Comment:
I would have written something like:`onSelect({ ...selected, from: date})` and `onSelect({ ...selected, to: date})` but I can see it's not working because `selected` is 
```
type DateRange = {
    from: Date | undefined;
    to?: Date | undefined;
};
```

But I think we could make our own type like this:
```
type DateRange = {
    from?: Date;
    to?: Date;
};
```
I think it's more appropriate, don't you think?

Reply:
Yes it would be better to not relying on `DateRange` type from the lib.
- it's easier to read in the handlers.
- we quickly get details about the type.

Thanks!